test/skimNtuple2018.cpp

```diff
@@ -2309,6 +2309,91 @@ int main (int argc, char** argv)
       float idAndIsoAndFakeSF_deep_pt = 1.0; // use this for DeepTauV2p1 pt dependent + e/mu->tauh fake SF
       float fakeRateSF_MVA  = 1.0;           // use this for e/mu->tauh fake SF MVA
       float fakeRateSF_deep = 1.0;           // use this for e/mu->tauh fake SF DeepTau
+       
+      float idAndIsoAndFakeSF_deep_pt_tau_pt20to25_up   = 1.0;
```

Comment:
maybe it's better to use the names suggested on the twiki: `idAndIsoAndFakeSF_tauid_pt20to25`...